ruby_review_week_four.rb

```diff
@@ -2,36 +2,136 @@
 
 # 1. What is a variable?
 
+#Variables are the things we store our data (strings,integers,booleans) in
+
 # 2. What Ruby method can you use to make a loop from an array?
 
+# In Ruby we would use a the .each method to interate over elements of an array.
+
 # 3. In Ruby, what are booleans?
 
+#Booleans in Ruby are a binary data type. They are either True or False.  
+
 # 4. How do classes relate to objects?
 
+# Classes are the blueprints that are used to create objects and determine their attributes 
 # 5. What is an instance variable?
 
+# An instance variable is 
+
+# An instance variable is a variable determined in a class. It has no scope outside that class. 
```

Comment:
true... i also would have liked you to use the word `attributes` in your definition, as in an instance variable is an attribute for an object